ms2deepscore/train_new_model/inchikey_pair_selection.py

```diff
@@ -349,12 +351,6 @@ def select_balanced_pairs(available_pairs_for_bin_matrix: np.ndarray,
                 heapq.heappush(available_inchikey_indexes,
                                (inchikey_counts[inchikey_with_lowest_count], inchikey_with_lowest_count))
 
-            # Also check if the second inchikey needs to be added back to the heap
```

Comment:
@florian-huber I think this can be removed, since the second inchikey of the pair is never removed from the available_inchikey_indexes, so it also does not have to be added back in. 
But please double check, since I did not work with heapq before.